stale-blocks.csv

```diff
@@ -34,8 +35,10 @@ height,hash,header
 849233,0000000000000000000020ea4b48f075a066131024c242b6f83eb20724c642d1,00401326836dc1fd3d5b80eea8546e52ca1d532cb16e3623937b020000000000000000001800e7811aa3c25e7623178b1035ce1955d8f9b5eb2a372fb93fd7a10a34591081d87866255d031769705114
 848968,000000000000000000006aade64c55965be1542f138da658dcdb93005e5c486b,0060c52e6d1ca822fe8a1737c2b529e11430215771304c501edf00000000000000000000b9029934d1e89e8e1519ce26779419acb1ff96868b76bea7f604d5fe2eab309634277666255d0317a69335b2
 848860,000000000000000000003f620e65976fc1768e6f81705cc7fdf25cfcb421825a,0000002682cf204cd1615b70a1de83aaec411fdb005a6004f7c701000000000000000000f92440cbf1cc9d5a4c3fe9c289270ed0351e8ceb978fdd09aebd3410bac2c67d9c237566255d031771f431a1
+848485,00000000000000000000953f5aac97508b1d9ed2cda5fb3d76ebccf3e6ed3141,0040f625d0bbfbc9f0d5acaf33ccaa92b5ba6cdb14ae5b3dd7f601000000000000000000c608be95434ccb4a7400305ce1bc1dcaca43c4fbf4322a66ef1001c9eb2fae751e957166b85c031748c3b55f
```

Comment:
Hm this block is not stale: https://mempool.space/block/00000000000000000000953f5aac97508b1d9ed2cda5fb3d76ebccf3e6ed3141

I'm not sure why the CI didn't catch this..